index.html

```diff
@@ -290,6 +111,50 @@ <h2 class="title1"> Contact me </h2>
         </div>
       </section>
   <!-- End Contact Form  -->
+  <!-- Popup section -->
+        <div class="popuphidden hidden">
+            <div class="popupcard">
+                <div class="pophead">
+                  <img class="popclose" id="closeModal" src="./img/close.png">
+                    <h2 class="poptitle"> Tonic </h2>
+                    <div class="popdetail">
+                        <p class="company">CANOPY</p>
+                        <img src="./img/Counter.png" alt="">
+                        <p class="jobdescription">Back End Dev</p>
+                        <img src="./img/Counter.png" alt="">
+                        <p class="year">2015</p>
+                    </div>
+                </div>
+                <div class="popimg">
+                    <img class="popimge">
+                </div>
+                <div class="sect">
+                <div class="popdescription">
+                    <p class="popdescrip"> Bla bla bla</p>
+                </div>
+                <div class="popbuttons">
+                    <div class="poptec">
+                        <ul class="popsmallbuttons">
+                            <li><button class="pop-button-small">html</button></li>
+                            <li><button class="pop-button-small">css</button></li>
+                            <li><button class="pop-button-small">JavaScript</button></li>
+                        </ul>
+                    </div>
+                    <div class="modal-buttons">
+                        <a href="#" class="btn img-btn live">
+                                   See live
+                        <img src="./img/icon.png" alt="See live">
+                        </a>
+                          <a href="#" class="btn img-btn source">
+                            See Source
+                         <img src="./img/vector-github.png" alt="See source" class="source-image">
+                         </a>
+                    </div>
+                </div>
+            </div>
+            </div>
+        </div>
```

Comment:
Like the previous code reviewers requested please stick to the figma design. My advise is you carefully inspect elements in the template and kindly pay attention to detail you design doesnt have borders and also its missing alot of features which ill point out using an illustration of a diagram.

This is your design:

![screen1](https://user-images.githubusercontent.com/79506105/189412139-bddcdfaa-5425-4bfc-9ebd-9fce4f5f65c2.png)

Please follow this design guideline on figma:

![Screen 2](https://user-images.githubusercontent.com/79506105/189414826-581c534f-dcb2-49f4-bed1-c675407fcc7b.png)
Your template should be almost exactly as the figma template you picked  please pay attention to detail.